services/user/App1/src/App.tsx

```diff
@@ -0,0 +1,27 @@
+import "./App.css";
+import { connect } from "@psibase/plugin";
```

Comment:
This should come from `@psibase/common-lib` or `/common`

Reply:
Why can't we use the external package if other app developers are gonna use it?

Reply:
I just meant that this API is part of the common libraries. And since supervisor does more than just plugin communication, I think it should be called "common-lib" or "common" rather than "plugin"